README.md

```diff
@@ -72,6 +72,11 @@ https://docs.github.com/en/issues/trying-out-the-new-projects-experience/about-p
 - `bug-fix-NUMBER-OF-ASSOCIATED-ISSUE`
 - `feature-FEATURE-NAME`
 
+### Versioning
+We have
+- semver.org as our standard
+- as pre-release: alpha and beta versions 
+- for bugfixes: x.x.bugfix[Name]
```

Comment:
We could write even a little more. I have written a few sentences for a client project a few days ago, please remind me to attach it here

Reply:
we will merge right now and will improve in the future